tests/cpp/multidevice.h

```diff
@@ -39,6 +39,12 @@ class MultiDeviceTest : public NVFuserTest {
       int64_t axis,
       const DeviceMesh& mesh);
 
+  // Validate the outputs of a fusion against expected outputs.
+  void validate(
```

Comment:
```suggestion
  static void validate(
```
IIUC, it doesn't need member access.